src/tutorials/ex3.c

```diff
@@ -0,0 +1,186 @@
+
+static char help[] = "Solves a tridiagonal system with lower bound specified as an inequality constraint.\n\
+Sovles finite differences discretizatio of:\n\
+-u''(x) = -15,  x in [0,1]\n\
+u(0) = u(1) = 0\n\
+s.t. u(x) >= sin(4*pi*x -pi/6)/2 -2\n\
+Based on ex1.\n\
+Input parameters include:\n\
+  -n <mesh_n> : number of mesh points\n\
+  -sol        : view and draw the solution vector\n\
+  -draw_puase : number of seconds to pause, -1 implies until user input, see PetscDrawSetPause()\n";
```

Comment:
```suggestion
  -draw_pause : number of seconds to pause, -1 implies until user input, see PetscDrawSetPause()\n";
```
or should be removed completely?
```suggestion
```

Reply:
resolved by bed115809a9f5c3372d35e6bb45d25f6cb977ff9